src/splitseq_mkref.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+#USAGE: ./splitseq_mkref.sh "genome names" "fasta files" "gtf files"
+
+genome_names=$1  #space-delimited, e.g. "hg38 mm10"
+fastas=$2        #space-delimited, e.g. "hg38.fa mm10.fa"
+gtfs=$3          #space-delimited, e.g. "hg38.gtf mm10.gtf"
+
+splitseq_venv="/Genomics/grid/users/rleach/local/splitseq"
```

Comment:
Consider creating an `environment.yml` file that creates a conda env with all required deps. This would be a bit more portable.

Reply:
I tried `conda env export > splitseq.yml`, but it has `prefix: /Genomics/grid/users/rleach/local/miniconda2/envs/splitseq` in the resulting file.  I'll see if I can figure out how to have that omitted tomorrow.

Reply:
For documentation pruposes, this was informative: https://github.com/conda/conda/issues/4339

Reply:
I almost always create the `environment.yml` by hand.

Reply:
Also, I didn't see the conda env file in the PR, did you mean to omit it? Would be useful I think.

Reply:
I haven't pushed the changes yet. I have implemented them locally.  Just testing to confirm the scripts still work with the changes.